cfg/CFG.cc

```diff
@@ -22,97 +32,161 @@ BasicBlock *CFG::freshBlock(int outerLoops, int rubyBlockId) {
     return r.get();
 }
 
+void CFG::enterLocalInternal(core::LocalVariable variable, LocalRef &ref) {
+    ENFORCE_NO_TIMER(!this->localVariablesFrozen);
```

Comment:
Would it be worth putting this ENFORCE in `enterLocal` instead? Is there a path where `enterLocalInternal` would be called when the CFG is frozen that doesn't also go through `enterLocal`?

Reply:
Good point; I think no.

Reply:
We discussed in DM and this placement is OK.